src/rules/indentRule.ts

```diff
@@ -50,7 +50,13 @@ class IndentWalker extends Lint.RuleWalker {
         }
         var scanner = ts.createScanner(ts.ScriptTarget.ES5, false, node.text);
         var lineStarts = node.getLineStarts();
+        var endOfComment = -1;
```

Comment:
This branch is behind `master`, you'll need to rebase to pick up some new linting rules. For example, we've turned on the `no-var-keyword` rule, so this should be a `const.